CorsixTH/Src/th_map.cpp

```diff
@@ -1048,31 +1086,14 @@ void level_map::draw(render_target* pCanvas, int iScreenX, int iScreenY,
         pItem->draw_fn(pItem, pCanvas, itrNode.x(), itrNode.y());
```

Comment:
You're no longer drawing the earlyEntities on a previous reverse pass... I suspect there might be fallout, need to pay attention to interactions closely.

Reply:
Random thought ... should we make a test suite? We could render some simple map test cases constructed to exercise various edge conditions. It would need some test image sprites so that it could render those into an image and compare to a reference screenshot.

This would help with confidently landing rendering changes. Maybe I'm overengineering this though :-).

Reply:
It would help a lot; not a small ask though?

Reply:
Yeah, it would be a large undertaking. I'm not suggesting we block this on it just maybe something to plan. It's challenging to have to worry about a bunch of cases you don't know about. A somewhat manual way of verifying might be to look at the bugs which were fixed by the change in draw order - if they have save files attached it could be an easy test.